resources/qml/Cura.qml

```diff
@@ -283,7 +283,7 @@ UM.MainWindow
             Menu
             {
                 id: plugin_menu
-                title: catalog.i18nc("@title:menu menubar:toplevel", "&Toolbox")
+                title: catalog.i18nc("@title:menu menubar:toplevel", "Marke&tplace")
```

Comment:
Typo?

Reply:
&t - it is a keyboard shortcut

Reply:
Mm it's kinda weird to have a keyboard shortcut that is not the first letter of the word (like we have in all other places).

Reply:
I wanted to keep the shortcut the same so it wouldn't conflict with other shortcuts

Reply:
This type of shortcut is only used while the menu is open, and since there is only a single menu item in the menu, there is no conflict possible.